src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/register/DetachedRegisterPage.java

```diff
@@ -0,0 +1,49 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth.register;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.DetachedSignInPage;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.DetachedSignInPage'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)